packages/react-components/review/react-components.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="react" />
```

Comment:
how come we are seeing these removals in this PR as well?

@PorterNan  any ideas here?